test/integration/setting_js_test.rb

```diff
@@ -29,4 +29,77 @@ class SettingJSTest < IntegrationTestWithJavascript
     assert_index_page(settings_path, "Settings", false, true, false)
     assert page.has_link?("My Pretty Setting Label", :href => "#category_label_test_settings_tab")
   end
+
+  test "general headers values" do
+    visit settings_path
+
+    assert_text 'Name'
+    assert_text 'Value'
+    assert_text 'Description'
+  end
+
+  test "edit single arrays values" do
+    visit settings_path
+    click_button(['#http_proxy_except_list'])
```

Comment:
It may have been a bad suggestion in my earlier review, but I think this may actually work since it should look for IDs too. However, I haven't tested it myself.
```suggestion
    click_button('http_proxy_except_list')
```